seqr/models.py

```diff
@@ -1111,6 +1111,7 @@ class RnaSeqSpliceOutlier(DeletableSampleMetadataModel):
         ('*', 'Any direction'),
     )
 
+    rank = models.IntegerField(null=True)
```

Comment:
this should never be null

Reply:
Okay, I thought we should support existing data in the database before the migration.

Reply:
There shouldn't be any of this data in the database yet, and if there was it would be broken without this field so we would want the migration to fail